docs/bcfusion/validators/staking.md

```diff
@@ -0,0 +1,133 @@
+# Staking
+
+BNB smart chain (BSC) is a Proof-of-Staked-Authority (PoSA) blockchain, which means that staking is one of the most
+important parts of the system. [BEP-294](https://github.com/bnb-chain/BEPs/pull/294) introduces the new native staking
+mechanism after BNB chain fusion, which has several improvements:
+
+- Users are able to participate in staking on the BSC using BNB in BEP20 format.
+- Staking credit, as the proof of BNB staked to a specified validator, cannot be transferred. Different validators issue
+  different staking credits.
+- The rewards for staking can be claimed more quickly.
+
+In this section, we will explain the basic concepts and operations of staking on the BSC.
+
+## Validator Set
+
+The validator set is the group of nodes that are responsible for validating transactions and producing blocks on the
+BSC. The validator set is determined by the amount of staking credit each validator has, which reflects the amount of
+BNB staked by the validator and its delegators. The top 45 validators with the most staking credit are selected as the
+active validator set, and they take turns to propose and vote on blocks. The rest of the validators are in the standby
+validator set, and they can join the active validator set if their staking credit increases or if some active validators
+drop out.
+
+The validator set is updated every 24 hours, based on the latest staking information. Validators can join or leave the
+validator set by creating or editing their validator information. Validators can also be removed from the validator set
+by slashing, which is a penalty for misbehaving or being offline.
+
+## Validator Operations
+
+Validators are the nodes that run the BNB smart chain software and participate in the consensus process. Validators need
+to have a minimum amount of BNB staked to their own validator address, and they can also accept delegations from other
+BNB holders who want to stake with them. Validators earn rewards from the transaction fees, and
+they share a portion of their rewards with their delegators.
+
+### Create Validator
+
+To create a validator, a BNB holder needs to send a `CreateValidator` transaction to the BNB smart chain,
+with minimum amount of BNB that the validator needs to stake to their own validator
+address (2000 BNB), specifying the following information:
+
+- **Operator address**: The address of the validator, which will receive the staking credit and the rewards.
+- **Consensus address**: The consensus address of the validator's node.
+- **Vote Address**: The address for participating fast finality voting.
+- **BLS Proof**: The data to prove that the validator owns the vote address.
+- **Commission rate**: The percentage of the rewards that the validator will keep for themselves, and the rest will be
```

Comment:
The commission info has three params: commissionRate, maxRate, maxChangeRate

Reply:
updated